index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"/>
+        <title>Setup & Mobile Portfolio Skeleton</title>
+</head>
+<body>
+    <main>
+        <header>
+            <nav class="navbar">
+            <h1>My Logo</h1>
+            <img class="hamburger" src="images/Union.png" alt="">
+            </nav>
+        </header>
+        <section class="welcome-section">
+            <h2>I'm Emmanuel <br />
+                Glad to see you!</h2>
+                <p>I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <h3>LET'S CONNECT</h3>
+                <div class="social-icons">
+                <img src="images/social-media-icons.png" alt="">
```

Comment:
- [ ] The list of icons should be an `img` element. Please kindly put each of the list of icons as an unordered list using `ul`. See hint below

```
<ul>
  <li><a><img src="" alt=""/></a></li>
</ul>
```